cros_gralloc/cros_gralloc_helpers.cc

```diff
@@ -138,8 +141,8 @@ uint64_t cros_gralloc_convert_usage(uint64_t usage)
 	handle_usage(&usage, GRALLOC_USAGE_HW_RENDER, &use_flags, BO_USE_RENDERING);
 	handle_usage(&usage, GRALLOC_USAGE_HW_2D, &use_flags, BO_USE_RENDERING);
 	/* HWC wants to use display hardware, but can defer to OpenGL. */
-	handle_usage(&usage, GRALLOC_USAGE_HW_COMPOSER, &use_flags,
-		     BO_USE_SCANOUT | BO_USE_TEXTURE);
+	handle_usage(&usage, GRALLOC_USAGE_HW_COMPOSER, &use_flags, BO_USE_TEXTURE);
+	handle_usage(&usage, GRALLOC_USAGE_PRIVATE_1, &use_flags, BO_USE_SCANOUT);
```

Comment:
I haven't a clue about why what is happening here and this change is required, so please provide some background.